conan_support.cmake

```diff
@@ -183,3 +186,91 @@ macro(conan_provide_dependency package_name)
     endif()
     find_package(${ARGN} BYPASS_PROVIDER)
 endmacro()
+
+
+function(conan_configure)
```

Comment:
It is not very clear the purpose of this generalization, with management of all arguments, but that might not be necessary most of them?